Source/Core/InputCommon/ControllerInterface/Touch/Touchscreen.h

```diff
@@ -0,0 +1,65 @@
+// Copyright 2008 Dolphin Emulator Project
+// Licensed under GPLv2+
+// Refer to the license.txt file included.
+
+#pragma once
+
+#include "InputCommon/ControllerInterface/ControllerInterface.h"
+#include "InputCommon/ControllerInterface/Touch/ButtonManager.h"
+
+namespace ciface::Touch
+{
+class Touchscreen : public Core::Device
+{
+private:
+  class Button : public Input
+  {
+  public:
+    std::string GetName() const;
+    Button(int padID, ButtonManager::ButtonType index) : _padID(padID), _index(index) {}
```

Comment:
Can we use this opportunity and address the naming convention differences in this file? Members/locals/parameters should be `snake_case` in general.
```suggestion
    Button(int pad_id, ButtonManager::ButtonType index) : m_pad_id(pad_id), m_index(index) {}
```